vnext/src/Libraries/Components/Touchable/TouchableOpacity.windows.js

```diff
@@ -19,6 +19,7 @@ import type {ViewStyleProp} from 'react-native/Libraries/StyleSheet/StyleSheet';
 import flattenStyle from 'react-native/Libraries/StyleSheet/flattenStyle';
 import Platform from '../../Utilities/Platform';
 import * as React from 'react';
+import {off} from 'process';
```

Comment:
what's this?

Reply:
honestly, not sure when that got there, now removed!